site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -348,6 +348,37 @@ Workloads as possible, preferring Workloads with these characteristics:
 - Workloads with the lowest priority.
 - Workloads that have been admitted more recently.
 
+## FlavorFungibility
```

Comment:
Add a note in the `Flavors and borrowing semantics` that some semantics can be modified based on the `flavorFungibility` field, and add a link to this section.